src/App.css

```diff
@@ -1,6 +1,364 @@
+  font-family: 'Roboto Slab', serif;
+  font-weight: 100;
+  font-size: 0.813rem;
+  color: white;
+  background-color: #0290ff;
+  padding: 0.4rem 1.4rem;
+  width: fit-content;
+  border-radius: 3px;
+}
+
+.addBookContainer {
+  padding: 1rem 4%;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.addNewBook {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  font-size: 1.25rem;
+  color: #888;
+  margin-top: 0.5rem;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 0.8rem;
+}
+
+input,
+select {
+  border: 1px solid #e8e8e8;
+  border-radius: 4px;
+  padding: 0.7rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+}
+
+::placeholder {
+  color: #c4c4c4;
+}
+
+.catContainer {
+  padding: 1rem 4%;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  text-align: center;
+}
+
+.catContainer button {
+  font-family: 'Roboto Slab', serif;
+  font-weight: 100;
+  font-size: 0.813rem;
+  color: white;
+  background-color: #0290ff;
+  padding: 0.4rem 1.4rem;
+  width: fit-content;
+  border: none;
+  border-radius: 3px;
+  align-self: center;
+}
+
+.statusMsg {
+  font-family: 'Montserrat', sans-serif;
+}
+
+/* -------------------------------------- */
+
+@media screen and (min-width: 768px) {
+  .headerWrapper {
+    padding: 2rem 6%;
+  }
+
+  .headerContainer {
+    gap: 2.7rem;
+  }
+
+  .logo {
+    font-size: 1.875rem;
+  }
+
+  .nav-links {
+    gap: 2.7rem;
+    font-size: 0.85rem;
+    letter-spacing: 1.9px;
+  }
+
+  .userIcon {
+    width: 3.2rem;
+    height: 3.2rem;
+    font-size: 1.5rem;
+  }
+
+  .bookCardWrapper {
+    padding: 2.5rem 6%;
+  }
+
+  .bookCardContainer {
+    display: grid;
+    grid-template-columns: 1.65fr 2fr;
+  }
+
+  .column-2 {
+    display: flex;
+    gap: 11%;
+    align-items: center;
+  }
+
+  form button {
+    width: 16%;
+    height: 2.813rem;
+  }
+
+  .line-2 {
+    width: 1px;
+    height: 4.375rem;
+    border: solid 1px #e8e8e8;
+    margin: 0;
+  }
+
+  .updateProgress {
+    padding: 0.5rem 1.9rem;
+  }
+
+  .addBookContainer {
+    padding: 1rem 6% 5rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+  }
+
+  form {
+    display: flex;
+    flex-direction: row;
+    gap: 2.8%;
+  }
+
+  input,
+  select {
+    border: 1px solid #e8e8e8;
+    border-radius: 4px;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 1rem;
+    width: 28%;
+    height: 2.813rem;
+  }
+
+  ::placeholder {
+    color: #c4c4c4;
+  }
+
+  .catContainer {
+    padding: 1rem 4%;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    text-align: center;
+  }
+
+  .catContainer button {
+    font-family: 'Roboto Slab', serif;
+    font-weight: 100;
+    font-size: 0.813rem;
+    color: white;
+    background-color: #0290ff;
+    padding: 0.4rem 1.4rem;
+    width: fit-content;
+    border: none;
+    border-radius: 3px;
+    align-self: center;
+  }
+
+  .statusMsg {
+    font-family: 'Montserrat', sans-serif;
+  }
 }
```

Comment:
- [ ] Good job so far :clap: Your app works as expected and your design is good. but please make sure that your `background color` with the background color of the original design instead of `white ` :+1: 

#### The original design
![Bookstore CMS](https://user-images.githubusercontent.com/6615394/201382877-e0e42d8b-07f8-41cf-bc4a-5de172904bae.jpg)